map_server/src/main.cpp

```diff
@@ -251,7 +252,7 @@ int main(int argc, char **argv)
 
   try
   {
-    MapServer ms(fname, res);
+    MapServer ms(private_nh, fname, res);
```

Comment:
Do we really need to pass in a special node handle? Isn't simply creating a node handle right after ros::init() above sufficient, such that we still have at least one NodeHandle in scope inside the catch()?

Reply:
I thought it best of initializing and sharing a single NodeHandle instead, but sure, I could switch to that If you prefer.

Reply:
Done